pkg/logs/input/journald/tailer.go

```diff
@@ -13,11 +13,12 @@ import (
 	"io"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Comment:
nit: I would have placed this import in a new section bellow journald but move pkg/logs